### PR TITLE
demo: add command to create demo records and fix prod setup

### PR DESCRIPTION
**Pull request by ppanero**
Changes:

* Executes setup commands inside the web_api container for `prod` version.
* Differences between docker-compose lower and greater than 1.21 (special characters accepatance).
* Adds command for data population (WIP in prod version)

### CODE REVIEW

File: invenio_cli/helpers/scripts.py [hunk 1]
```diff
@@ -295,4 +295,31 @@ def server(dev=True, docker_helper=None, app_name='invenio-rdm',
     if dev:
         _server_dev(docker_helper, loglevel, logfile, verbose)
     else:
-        _server_prod()
+        click.secho("Starting docker contianers. " +
```
Comment (fenekku) [inline invenio_cli/helpers/scripts.py]:
```suggestion
        click.secho("Starting docker containers. " +
```

File: invenio_cli/cli.py [hunk 2]
```diff
@@ -332,3 +332,22 @@ def upgrade(log_level, verbose):
...
+
+@cli.command()
+@click.option('--dev/--prod', default=True, is_flag=True,
+              help='Which environment to build, it defaults to development')
+@click.option('--log-level', required=False, default='warning',
+              type=click.Choice(list(LEVELS.keys()), case_sensitive=False))
+@click.option('--verbose', default=False, is_flag=True, required=False,
+              help='Verbose mode will show all logs in the console.')
+def demo(dev, log_level, verbose):
+    """Upgrades the current application to the specified newer version."""
```
Comment (fenekku) [inline invenio_cli/cli.py]:
```suggestion
    """Populates instance with demo records."""
```

File: invenio_cli/helpers/scripts.py [hunk 3]
```diff
@@ -233,6 +221,62 @@ def setup(dev=True,  force=False, docker_helper=None,
...
+        click.secho("Purging queues...", fg="green")
+        docker_helper.execute_cli_command(
+            project_shortname, 'invenio index queue init purge')
+
+    click.secho("Creating database...", fg="green")
+    docker_helper.execute_cli_command(
+        project_shortname, 'invenio db init create',)
+    click.secho("Creating indexes...", fg="green")
+    docker_helper.execute_cli_command(
+        project_shortname, 'invenio index init --force')
```
Comment (fenekku) [inline invenio_cli/helpers/scripts.py]: Should this `--force` be here?

Comment (ppanero) [reply invenio_cli/helpers/scripts.py]: True, the is a use case for it (adding new schemas) but is not our case.

**Pull request merged**